py/selenium/webdriver/remote/webelement.py

```diff
@@ -235,20 +235,13 @@ def send_keys(self, *value) -> None:
     @property
     def shadow_root(self) -> ShadowRoot:
         """Returns a shadow root of the element if there is one or an error.
-        Only works from Chromium 96 and Firefox 96 onwards. Previous versions
-        of browsers will throw an assertion exception.
+        Only works from Chromium 96, Firefox 96, and Safari 16.4 onwards.
+        Previous versions of browsers will throw an assertion exception.
```

Comment:
This line can be removed as it is no longer true.

Reply:
"Previous versions of browsers will throw an assertion exception"?